src/nbl/video/device_capabilities/device_features.json

```diff
@@ -212,16 +212,16 @@
                 {
                     "type": "bool",
                     "name": "fragmentStoresAndAtomics",
-                    "value": false,
+                    "value": true,
                     "expose": "MOVE_TO_LIMIT",
                     "comment": [
-                        "ROADMAP 2022 no supporton iOS GPUs"
+                        "ROADMAP 2024 no supporton iOS GPUs"
                     ]
                 },
                 {
                     "type": "bool",
                     "name": "shaderTessellationAndGeometryPointSize",
-                    "value": false,
+                    "value": true,
                     "expose": "MOVE_TO_LIMIT",
```

Comment:
this is a limit, leave the default value as is (the default values are "worst case" basically)